arch/arm/boot/dts/bcm2710.dtsi

```diff
@@ -97,6 +157,9 @@
 
 &intc {
         compatible = "brcm,bcm2836-armctrl-ic";
```

Comment:
It's not your fault that this node uses spaces for indentation where everything else uses tabs, but while you are in this area you could do us a favour by fixing it.